locations/spiders/mcdonalds_it.py

```diff
@@ -7,13 +7,16 @@
 from locations.categories import Categories, apply_category
 from locations.dict_parser import DictParser
 from locations.hours import OpeningHours
+from locations.settings import DEFAULT_PLAYWRIGHT_SETTINGS
+from locations.user_agents import BROWSER_DEFAULT
 
 
 class McdonaldsITSpider(scrapy.Spider):
```

Comment:
Playwright spiders should ideally now have the following class definition:

```
from locations.playwright_spider import PlaywrightSpider
...
class McdonaldsITSpider(PlaywrightSpider):
```

No need for `is_playwright_spider = True` either.